### ISSUE TITLE
binary_crossentropy used with > 2 labels

### ISSUE THREAD

**Issue by i2infinity** [IBM/audioset-classification#10]
https://github.com/IBM/audioset-classification/blob/f5ef8bfb9c75540b42854a61d5f5bebd6751f199/audioset_classify/keras/core.py#L122

Shouldn't we be using a `categorical_crossentropy` loss function? The audio set data has > 2 labels in many cases.

Also, I am not seeing a good accuracy on using `categorical_crossentropy` loss function. Wondering if a CNN help here?

**Comment by i2infinity** [comment]
Nevermind... I think `binary_crossentropy` is the right choice here
